Makefile

```diff
@@ -47,6 +47,11 @@ $(call help,make lint,"lint the code and print any warnings")
 lint: python
 	@pyenv exec tox -qe lint
 
+.PHONY: typecheck
+$(call help,make typecheck,"Type check the code and print any warnings")
+typecheck: python
+	@pyenv exec tox -qe typecheck
+
```

Comment:
`make template` will remove this. We'll need to either add it to the cookiecutter (if we want all our apps to have it) or move it into [the `lms.mk` file](https://github.com/hypothesis/lms/blob/main/lms.mk) (which `Makefile` imports and which the cookiecutter doesn't touch)